src/components/Picker/Picker.driver.ts

```diff
@@ -10,6 +10,8 @@ export interface PickerDriver extends BaseUniDriver {
   clickOnPrev(): Promise<void>;
   hasDisablePrev(): Promise<boolean>;
   hasDisableNext(): Promise<boolean>;
+  hasNextAriaLabel(nextAriaLabel): Promise<boolean>;
+  hasPrevAriaLabel(prevAriaLabel): Promise<boolean>;
```

Comment:
I think instead of having a boolean method, we can just return the ariaLabel:
```suggestion
  getNextAriaLabel(nextAriaLabel): Promise<string | undefined>;
  getPrevAriaLabel(prevAriaLabel): Promise<string | undefined>;
```
and then just return 
```js
return base.$(`[data-hook=my-next-button]`).attr('aria-label')
```